src/hooks/useItems.tsx

```diff
@@ -71,6 +80,8 @@ const convertItemsToNodes = (items: ItemType[], props: Props) => {
         collapsible={mergeCollapsible}
         onItemClick={handleItemClick}
         destroyInactivePanel={mergeDestroyInactivePanel}
+        headingLevel={headingLevel}
+        id={id ? `${id}__item-${index}` : undefined}
```

Comment:
You can use `rc-util` useId hooks in Panel. It support to use user pass id and fallback with ReactID if not have.